src/mono/CMakeLists.txt

```diff
@@ -952,6 +952,19 @@ endif()
 
 include_directories("${CLR_SRC_NATIVE_DIR}")
 
+include(FetchContent)
+
+FetchContent_Declare(
+  dnmd
+  GIT_REPOSITORY https://github.com/AaronRobinsonMSFT/DNMD
```

Comment:
This will introduce C++ runtime dependency in mono and we are ok with this, neither we can lower the dependency in DNMD to just use the plan C, right?

Reply:
DNMD core libraries (dnmd and dnmd::pdb) only use C. The dnmd::interfaces library uses C++, but that is intended for use in CoreCLR, not here.

Reply:
I wonder if this would block or be a pain of offline build.

Reply:
This PR is just for testing. If we were to actually ship this, we would likely integrate DNMD into the repo or use dependency flow to pass the dependencies.